api/api.yaml

```diff
@@ -191,6 +191,26 @@ paths:
           $ref: '../result/request.yaml#/components/responses/BadRequest'
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
+  
+  /control/actions:
+    post:
+      tags: ['Control']
+      operationId: set_action
```

Comment:
should be set_device_state to align with the config device model

Reply:
modified set_action --> set_device_state